service/discovery/azure/storage.go

```diff
@@ -107,8 +138,14 @@ func (d *azureStorageDiscovery) discoverStorageAccounts() ([]voc.IsCloudResource
 		return nil, err
 	}
 
+	// Discover backup vaults
+	err := d.azureDiscovery.discoverBackupVaults()
```

Comment:
This is intentionally called in the storage AND in the compute discovery, correct? Do we have a problem that the discovered vaults are then created twice?

Reply:
Ok I guess that's ok-ish for now.